backend/app/api/sms.py

```
@@ -0,0 +1,240 @@
+        SMSTemplate(
+            template="یادآوری: [نام حیوان] نژاد [نژاد] برای واکسن [نوع واکسن] در تاریخ [تاریخ]. کلینیک پارسیان",
+            variables=["[نام حیوان]", "[نژاد]", "[نوع واکسن]", "[تاریخ]"]
+        )
+    ]
+    
+    return templates
+
+@router.delete("/archive/{sms_id}")
+async def delete_sms_archive(
+    sms_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)